src/Strongly.Options.SourceGenerators/Extensions/AssemblySymbolExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System.Linq;
+using Microsoft.CodeAnalysis;
+
+namespace Strongly.Options.SourceGenerators.Extensions;
+
+using static WellKnownNamings;
+
+public static class AssemblySymbolExtensions
+{
+    public static string GetModuleNameFromAttributeOrShortAssemblyNameByDefault(this IAssemblySymbol assembly)
```

Comment:
если вдруг где-то можно задокументировать что default это имя типа
```suggestion
    public static string GetModuleNameFromAttributeOrDefault(this IAssemblySymbol assembly)
```

Reply:
It's unclear from the method that we're using ShortAssemblyName in default case